src/reset.css

```diff
@@ -67,11 +67,11 @@ select {
   font: inherit;
 }
 
-/* Remove _all_ animations and transitions for people that prefer not to see them */
+/* Remove transitions for people that prefer not to see them */
+/* Animations should be handled manually */
 @media (prefers-reduced-motion: reduce) {
   * {
-    animation-play-state: paused !important;
```

Comment:
This should probably be replaced with:

```css
animation-duration: 0.01ms !important;
animation-iteration-count: 1 !important;
```

Reply:
That does seem like it would probably be safe with the `animationend` event, since those using infinite animations won't be listening to the event anyway

Reply:
Yeh for sure, it feels like a nice happy medium to me :)